modules/manage/pages/topic-iceberg-integration.adoc

```diff
@@ -0,0 +1,117 @@
+= Topics as Iceberg Tables
+:description: Learn how to integrate Redpanda topics with Iceberg.
+:page-context-links: [{"name": "Linux", "to": "manage:topic-iceberg-integration.adoc" } ]
+:page-categories: Management, High Availability
+
+
+The Redpanda integration with Apache Iceberg allows your topic data to be stored in the cloud in Iceberg's table format. This opens up immediate access to your streaming data for analytical systems such as data warehouses like RedShift, Snowflake, and Clickhouse, and big data processing platforms such as Apache Spark and Flink, without setting up and maintaining additional ETL pipelines. 
+
+== Prerequisites
+
+include::shared:partial$enterprise-license.adoc[]
+
+. To check if you already have a license key applied to your cluster:
++
+[,bash]
+----
+rpk cluster license info
+----
+
+. Install `rpk`.
+
+== Limitations
+
+* You can only enable the Iceberg integration when creating a new Redpanda topic. 
+* It is not possible to append or backfill data from Redpanda topics to an existing Iceberg table.
+* JSON schemas are not currently supported.
```

Comment:
Is Avro supported for beta?

Reply:
With the limitation that no single item in the schema be more than 128K.  It would be incredibly unusual, but not impossible, to find a schema with such an entry.